### PR TITLE
Update profile

### LINKED ISSUE #133: Update Profile (and set new password)
See own profile
Manage profile 
set new password

### PR DESCRIPTION
**Pull request by JuDithGreat**
first try to work on together, dear Jan :)

### CODE REVIEW

**Comment by CodeOneTwo** [comment]
Sehr nice... judithgreat. Huge fan of this pr. :smile:

File: frontend/src/app/navigation/navigation.component.html [hunk 1]
```diff
@@ -40,6 +40,10 @@
     <mat-icon>expand_more</mat-icon>
   </button>
   <mat-menu #menu="matMenu">
+    <button mat-menu-item (click)="userService.edit()">
```
Comment (CodeOneTwo) [inline frontend/src/app/navigation/navigation.component.html]: öööh nope, routerLink="/user/edit" or similar

File: frontend/src/app/user/register/register.component.ts [hunk 2]
```diff
@@ -102,4 +102,18 @@ const passwordStrengthValidator = (
         'Password must contain at least two of the following: numbers, lowercase letters, uppercase letters, or special characters.',
     };
   }
+
 };
+
+export interface IRegister {
```
Comment (CodeOneTwo) [inline frontend/src/app/user/register/register.component.ts]: This is not an interface

an interface defines the types:

```
interface IInterfaceName {
    gender: string;
    firstname: string;
...
}

File: frontend/src/app/user/edit/edit.component.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  });
+
+  constructor(
+    private userService: UserService,
+    private router: Router,
+    private snackBar: MatSnackBar
+  ) {}
+
+  ngOnInit(): void {
+    this.userService.get().subscribe((user) => {
```
Comment (CodeOneTwo) [inline frontend/src/app/user/edit/edit.component.ts]: this.userService.getProfile

**Comment by JuDithGreat** [comment]
Oh wow! I was almost there :O

**Pull request closed without merging**